provisioner/template.go

```diff
@@ -675,3 +676,20 @@ func certificateExpiry(certificate string) (string, error) {
 	}
 	return expiry.UTC().Format(time.RFC3339), nil
 }
+
+func sumQuantities(quantities ...string) (string, error) {
+	var result k8sresource.Quantity
+	for _, quantity := range quantities {
+		q, err := k8sresource.ParseQuantity(quantity)
+		if err != nil {
+			return "", err
+		}
+		if result.IsZero() {
```

Comment:
minor: `result.IsZero()` seems redundant as result is zero by default and Add should work

Reply:
Fixed